frontend/app/src/utils/permissions.ts

```diff
@@ -0,0 +1,46 @@
+export type tPermission = {
+  view: string;
+  create: string;
+  update: string;
+  delete: string;
+};
+
+export type uiPermission = {
```

Comment:
WDYT:
`tPermission` => `PermissionData`
`uiPermission` => `Permission`

Reply:
sure, it was just a draft for the naming